tests/snmpsim/cnpilotr_1.snmprec

```diff
@@ -0,0 +1 @@
+1.3.6.1.2.1.1.1.0|4|cnPilot R201 4.3.3-R4
```

Comment:
Please add both sysObjectID and sysDescr for snmprec files.

Reply:
Same situation as above. SysObjectID is a bust...

Reply:
@pheinrichs We need to include them otherwise our testing will be worthless.
Even (especially) if it just returns an empty string.